staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -315,9 +323,32 @@ func cleanVerb(verb string, request *http.Request) string {
 	if verb == "WATCHLIST" {
 		reportedVerb = "WATCH"
 	}
+	if verb == "PATCH" && request.Header.Get("Content-Type") == string(types.ApplyPatchType) {
+		reportedVerb = "APPLY"
```

Comment:
`APPLY` isn't an API server verb, and we're recording content type as well... why do we want to synthesize a verb here?

Reply:
I'm not a huge fan of munging this into the verb, but I am also hesitant to add another label to such a high volume metric. Guard this with the apply feature gate for now, and let's add this to the things to think about before beta